### PR TITLE
Link redirects

### PR DESCRIPTION
**Pull request by Solexi**
# Fix incorrect link for Suchet Dhindsa's wiki page

_Fixes the issue where clicking on Suchet Dhindsa's wiki page on the about page takes you to Justin Moore's wiki page. The link has been updated to the correct URL._

## How should this be tested?

1. Navigate to the about page.
2. Click on Suchet Dhindsa's wiki page link.
3. Verify that it redirects to the correct URL: https://iq.wiki/wiki/suchet-dhindsa-salvesen/

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/3061

### CODE REVIEW

**Comment by Tolu-Mals** [comment]
Can you check this out

Screenshot 2024-08-19 at 10 01 59 AM

**Comment by Tolu-Mals** [comment]
> Can you check this out
> 
> Screenshot 2024-08-19 at 10 01 59 AM

updating your branch with main will likely fix this

**Review by Softdev1 — APPROVED** [review]
this works very fine now 
image

File: src/components/CreateWiki/EditorModals/FeedbackModal.tsx
Comment (Tolu-Mals) [inline src/components/CreateWiki/EditorModals/FeedbackModal.tsx:1]: please remove this file

**Review by Tolu-Mals — APPROVED** [review]
Once you remove the file, feel free to merge.

**Pull request merged**